fuzzing/test_case_assertion_provider.go

```diff
@@ -44,6 +45,16 @@ func attachAssertionTestCaseProvider(fuzzer *Fuzzer) *AssertionTestCaseProvider
 // isTestableMethod checks whether the method is configured by the attached fuzzer to be a target of assertion testing.
 // Returns true if this target should be tested, false otherwise.
 func (t *AssertionTestCaseProvider) isTestableMethod(method abi.Method) bool {
+	// Do not test optimization tests
+	if utils.IsOptimizationTest(method, t.fuzzer.config.Fuzzing.Testing.OptimizationTesting.TestPrefixes) {
```

Comment:
I'm fine with this for now, but we'll probably allow this in the future just because on an API level we don't want to disallow any of this. 

Or in the least, we'd make sure they didn't have a shared test prefix or something, idk.

The latter statement I made is a bit dumber, but the former I think makes sense later. For now, it doesn't matter though, you can disregard, just wanted you to be aware and share any thoughts if you disliked that thought for the future. :)